examples/chart/teleport-cluster/charts/teleport-operator/templates/_helpers.tpl

```diff
@@ -0,0 +1,40 @@
+{{/*
+Create the name of the service account to use
+if serviceAccount is not defined or serviceAccount.name is empty, use .Release.Name
+*/}}
+{{- define "teleport-cluster.operator.serviceAccountName" -}}
+{{- coalesce .Values.serviceAccount.name .Release.Name -}}-operator
```

Comment:
Suffixing with -operator is weird in standalone deployemnts if the release name is `teleport-operator`, we end up with `teleport-operator-operator`. There might be something to do with a nameOverride.